corehq/apps/app_manager/forms.py

```diff
@@ -69,6 +69,11 @@ def clean_domain(self):
         domain_obj = Domain.get_by_name(domain)
         if domain_obj is None:
             raise forms.ValidationError("A valid project space is required.")
+        if toggles.MULTI_MASTER_BYPASS_VERSION_CHECK.enabled(domain):
+            raise forms.ValidationError("""
+                Copying an app to a domain the uses multi-master linked apps and bypasses
+                the minimum CommCare version check requires developer intervention.
```

Comment:
check which* requires?

Reply:
that one's intentional

Reply:
Hmm, i would see this as

```
Copying an app to a domain *that* uses multi-master linked apps *which* bypasses
the minimum CommCare version check*. This* requires developer intervention.
```

Reply:
I don't think that makes sense, let's offline if you think the existing wording doesn't make sense.